tests/dashboard.spec.ts

```diff
@@ -21,7 +22,9 @@ test.describe('on dashboard screen', () => {
     const isSomeOfAstarEndpoints = providerEndpoints[endpointKey.ASTAR].endpoints.some(
       (it) => it.endpoint === selectedEndpoint
     );
+    const isNotLightClient = checkIsLightClient(selectedEndpoint);
```

Comment:
Looks like Not is not needed in the variable name. If you name variable like that I will expect the following condition `expect(isNotLightClient).toBe(true);`
just rename the variable to `isLightClient` and everything will be fine :)

Reply:
Good catch! I've fixed it in [this](https://github.com/AstarNetwork/astar-apps/pull/820/commits/2aba3fccdc27e75fb4e818d48c26c037e1d421c8) commit.